### PR TITLE
Main screen

### LINKED ISSUE #7: Main screen UI
# REQUIREMENTS
Create main screen that shows either a list or a map

# DESIGN
Screen Shot 2020-12-03 at 11 42 54

# Acceptance Criteria
List all required acceptance criteria
- [ ] Create a screen that shows the available stores/restaurants
- [ ] Create a map (Not the functionality or API, just an image).
- [ ] Create a list (?)

### PR DESCRIPTION
**Pull request by tovl1**
# Description
This is related to issue #7
I create a new screen for the Main Screen.
* Create MainScreen.js
* Create MapStyle.js
* Create constant.js
* Add icons .png. The icons is not final design.

 # Testing
  
  ## Screenshots 
mainScreen

### CODE REVIEW

**Review by JordiGN — DISMISSED** [review]

**Review by Ocosmee — DISMISSED** [review]
Good Job

File: tribomobile/App.js [hunk 1]
```diff
@@ -7,19 +7,20 @@
...
 } from 'react-native';
 
-
 const App: () => React$Node = () => {
   return (
     <>
       <StatusBar barStyle="dark-content" />
       <SafeAreaView>
-        <Text>Hola Tribo App</Text>
+        {/* <Text>Hola Tribo App</Text> */}
```
Comment (OswaldoPineda) [inline tribomobile/App.js]: remove this line if it's not necessary

File: tribomobile/components/constant.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const Strings = 
+    {
+        food: 'Comida     ',
+        product: 'Productos     ',
+        service: 'Servicios     '
+    }
+;
+
+export default Strings;
```
Comment (OswaldoPineda) [inline tribomobile/components/constant.js]: add an extra line here

File: tribomobile/components/mainScreen.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+import food from '../assets/food.png';
+import store from '../assets/store.png';
+import user from '../assets/user1.png';
+import MapStyle from './MapStyle';
+import string from './constant';
+
+const MainScreen = () => {
+  function pressMenu() {
+    console.log('Se presiono el menu');
+  }
```
Comment (OswaldoPineda) [inline tribomobile/components/mainScreen.js]: is this method necessary?

File: tribomobile/components/MapStyle.js [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+      elementType: 'labels.text.fill',
+      stylers: [
+        {
+          color: '#9e9e9e',
+        },
+      ],
+    },
+  ];
+
+export default MapStyle;
```
Comment (OswaldoPineda) [inline tribomobile/components/MapStyle.js]: add an extra line here

**Review by Ing-Heber — DISMISSED** [review]

File: tribomobile/components/constant.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+const Strings = 
+    {
+        food: 'Comida     ',
+        product: 'Productos     ',
+        service: 'Servicios     '
+    }
+;
```
Comment (luisarmandodv) [inline tribomobile/components/constant.js]:
```suggestion
const Strings = {
  food: 'Comida',
  product: 'Productos',
  service: 'Servicios',
};
```

**Review by luisarmandodv — CHANGES_REQUESTED** [review]
all the rest of the code looks good

**Review by alcamk — DISMISSED** [review]
Good job!

File: tribomobile/components/mainScreen.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+import menuImage from '../assets/menuImage.png';
+import food from '../assets/food.png';
+import store from '../assets/store.png';
+import user from '../assets/user1.png';
+import MapStyle from './MapStyle';
+import string from './constant';
+
+const MainScreen = () => {
+  function pressMenu() {
+    console.log('Se presiono el menu');
```
Comment (LeyaimMagma) [inline tribomobile/components/mainScreen.js]: Remove console.log

File: tribomobile/components/mainScreen.js [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+        </TouchableOpacity>
+        <TextInput style={style.textinput} placeholder="Escribe tu dirección" />
+      </View>
+      <View elevation={5} style={[style.navDown, style.iconsDown]}>
+        <Image source={food} style={style.imagenServices} />
+        <Text style={style.textDown}>{string.food}</Text>
+        <Image source={store} style={style.imagenServices} />
+        <Text style={style.textDown}>{string.product}</Text>
+        <Image source={user} style={style.imagenServices} />
+        <Text style={style.textDown}>{string.service}</Text>
```
Comment (LeyaimMagma) [inline tribomobile/components/mainScreen.js]: can you refactor this?

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Comment by LeyaimMagma** [comment]
@oswaldotovar32  Resolve Conflicting files

File: tribomobile/components/mainScreen.js (hunk 6 above)
Comment (tovl1) [reply tribomobile/components/mainScreen.js]: Done! c:

File: tribomobile/components/MapStyle.js (hunk 4 above)
Comment (tovl1) [reply tribomobile/components/MapStyle.js]: done!

File: tribomobile/components/mainScreen.js (hunk 3 above)
Comment (tovl1) [reply tribomobile/components/mainScreen.js]: No, I'm testing for the method onpress={} in TouchableOpacity

File: tribomobile/components/constant.js (hunk 2 above)
Comment (tovl1) [reply tribomobile/components/constant.js]: Done!

File: tribomobile/App.js (hunk 1 above)
Comment (tovl1) [reply tribomobile/App.js]: Done!

File: tribomobile/components/mainScreen.js (hunk 7 above)
Comment (tovl1) [reply tribomobile/components/mainScreen.js]: Done!

**Review by JordiGN — DISMISSED** [review]
Nice, it is a really good first step, good job

**Review by develiasdaniel — DISMISSED** [review]

**Review by Ing-Heber — DISMISSED** [review]

**Review by JordiGN — DISMISSED** [review]

**Review by Ocosmee — DISMISSED** [review]
Good job!

**Review by alcamk — DISMISSED** [review]
Great job!

**Review by Ing-Heber — DISMISSED** [review]

**Review by JordiGN — APPROVED** [review]

**Review by LeyaimMagma — APPROVED** [review]

**Review by Ing-Heber — APPROVED** [review]

**Review by luisarmandodv — APPROVED** [review]

**Pull request merged**